Add some author guidelines (#69)
project-chip/matter-handbook: README.md

# matter-handbook

## Author guidelines
This is an informal set of guidelines for authors, where we can document the conventions that are being used in this handbook.

### Public information

This is a public website. Do not include any Alliance confidential information in these documents.

### Acronyms
- Write out acronyms on first use and include the acronym in parentheses
  - First use of a Three Letter Acronym (TLA) should look similar to this.
- Add new acronyms to the .wordlist.txt file for the spell checker
- Add new acronyms glossary if appropriate

### External links

> The handbook is meant to be used by a variety of users, not only CSA members. This may include adopter and associate members as well as companies considering membership.

Reviewer: ```suggestion
The handbook is meant to be used by a variety of users, not only Alliance members. This may include adopter and associate members as well as companies considering membership.
```

Please use the following guidelines for links:

> - When possible, link to public resources (CSA public website, open source github repositories on project-chip)

Author: Is that also something that needs to be added to the guidelines? Is CSA not something to be used? Is alliance preferred or CSA-IOT?

> - When possible, link to public resources (CSA public website, open source github repositories on project-chip)

Reviewer: Yes, please. I believe we are no longer using 'CSA' in public facing documents. Please use 'Alliance' instead.

# matter-handbook

## Author guidelines

> This is an informal set of guidelines for authors, where we can document the conventions that are being used in this handbook.

Reviewer 2: ```suggestion
This is an informal set of guidelines for authors, where we can document the conventions that are being used in this Handbook.
```

### Connectivity Standards Alliance Naming

> When referring to the Alliance, in the first reference use the full name as "Connectivity Standards Alliance (Alliance)". Use "Alliance" in subsequent references on the same page.

Author: ```suggestion
When referring to the Alliance, in the first reference use the full name as "Connectivity Standards Alliance (Alliance)". Use "Alliance" in subsequent references.
```

### Public information

This is a public website. Do not include any Alliance confidential information in these documents.

### Acronyms
- Write out acronyms on first use and include the acronym in parentheses
  - First use of a Three Letter Acronym (TLA) should look similar to this.
- Add new acronyms to the `.wordlist.txt` file for the spell checker

> - Add new acronyms glossary if appropriate

Reviewer 2: ```suggestion
- Add new acronyms to the Glossary if appropriate
```

### External links

> The handbook is meant to be used by a variety of users, not only Alliance members. This may include adopter and associate members as well as companies considering membership.

Reviewer 2: ```suggestion
The Handbook is meant to be used by a variety of users, not only Alliance members. This may include Adopter and Associate members as well as companies considering membership.
```

Please use the following guidelines for links:
- When possible, link to public resources (Alliance public website, open source GitHub repositories on project-chip)

> - If a page contains links to any non public resource it should include the following disclaimer at the top:

Reviewer 2: ```suggestion
- If a page contains links to any non-public resource it should include the following disclaimer at the top:
```

  - `> Some links on this page are only accessible to Alliance members.`

> - If a causeway link is required and the required resource is expected to be used by adopter members, ensure the resource is a sub-folder in the ["All Members" section](https://groups.csa-iot.org/wg/members-all/document). If the resource is expected to be accessible to adopter members and is not in that folder, consider moving it.

Reviewer 2: ```suggestion
- If a Causeway link is required and the required resource is expected to be used by Adopter members, ensure the resource is a sub-folder in the ["All Members" section](https://groups.csa-iot.org/wg/members-all/document). If the resource is expected to be accessible to Adopter members and is not in that folder, consider moving it.
```

> - Pages that contain resources that are specifically aimed at Participant and Promoter members (ex. private github repositories, access to documents in the Matter work group causeway) should include a disclaimer at the top of the file.

Reviewer 2: ```suggestion
- Pages that contain resources that are specifically aimed at Participant and Promoter members (for example, private GitHub repositories, or access to documents in the Matter work group causeway) should include a disclaimer at the top of the file.
```

> - Pages that contain resources that are specifically aimed at Participant and Promoter members (ex. private github repositories, access to documents in the Matter work group causeway) should include a disclaimer at the top of the file.

Reviewer 2: I think "Matter work group causeway" should be phrased differently but I'm not sure how.

...in the Matter working group in Causeway?  I don't know what specific group you're talking about.

Reviewer: "in the Matter working group in Causeway" Sounds good to me.

Author: changed.

> - Pages that contain resources that are specifically aimed at Participant and Promoter members (for example, private GitHub repositories, or access to documents in the Matter work group causeway) should include a disclaimer at the top of the file.

Author: ```suggestion
- Pages that contain resources that are specifically aimed at Participant and Promoter members (for example, private GitHub repositories, or access to documents in the Matter working group in Causeway) should include a disclaimer at the top of the file.
```
